### PR TITLE
html validator errors fixed in achievements.html

### PR DESCRIPTION
**Pull request by Castella1313**
issue solved #666 

**All the html validator errors have been fixed in the file "achievements.html**

**Screenshot:**
Screenshot 2020-05-04 at 2 07 44 PM

### CODE REVIEW

File: achievements.html [hunk 1]
```diff
@@ -1,45 +1,45 @@
...
 	<title>GirlScript Foundation-Chennai Chapter | Achievements</title>
-	<meta charset="utf-8">
 	<meta content="width=device-width, initial-scale=1" name="viewport" />
 	
 	<meta name="keywords"
-		content="tech events, coding competition, introduction to coding, computer coding for beginners, girls who code program, coding contest, coding competition, girls can code, introduction to programming, learn computer programming">
+		content="tech events, coding competition, introduction to coding, computer coding for beginners, girls who code program, coding contest, coding competition, girls can code, introduction to programming, learn computer programming"/>
 	<meta name="description"
-		content="This website is a base for Chennai chapter, and serves a template for other chapters. All the channel-specific tech events, coding competitions, and other resources are mentioned in this website.">
+		content="This website is a base for Chennai chapter, and serves a template for other chapters. All the channel-specific tech events, coding competitions, and other resources are mentioned in this website."/>
```
Comment (smaranjitghose) [inline achievements.html:11]: Please update this according to the Readme

File: achievements.html [hunk 2]
```diff
@@ -1,45 +1,45 @@
...
 	
-	<meta name="application-name" content="GirlScript Foundation-Chennai Chapter | Achievements">
+	<meta name="application-name" content="GirlScript Foundation-Chennai Chapter | Achievements"/>
```
Comment (smaranjitghose) [inline achievements.html:13]: GirlScript Website Boilerplate

File: achievements.html [hunk 3]
```diff
@@ -1,45 +1,45 @@
...
-	<meta name="robots" content="index,follow">
-	<meta name="Googlebot" content="index,follow">
+	<meta name="robots" content="index,follow"/>
+	<meta name="Googlebot" content="index,follow"/>
 	<meta name="AdsBot-Google" content="index,follow" />
 	
-	<meta name="designer" content="GirlScript Foundation-Chennai Chapter | Achievements">
-	<meta name="copyright" content="GirlScript Foundation-Chennai Chapter | Achievements">
+	<meta name="designer" content="GirlScript Foundation-Chennai Chapter | Achievements"/>
+	<meta name="copyright" content="GirlScript Foundation-Chennai Chapter | Achievements"/>
```
Comment (smaranjitghose) [inline achievements.html:22]: As mentioned above

File: achievements.html [hunk 4]
```diff
@@ -286,7 +285,7 @@ <h2 class="card-title">
 
 				<div class="project-card" onclick="displayCard(this)">
 					<div class="project-image-holder">
-						<img class="project-image" src="https://source.unsplash.com/300x225/?river" alt="river" aria-label="River/>
+						<img class="project-image" src="https://source.unsplash.com/300x225/?river" alt="river" aria-label=River/>
```
Comment (smaranjitghose) [inline achievements.html:288]: Make this a local asset

File: achievements.html [hunk 5]
```diff
@@ -1,45 +1,45 @@
 <!DOCTYPE html>
-<html>
+<html lang = "en">
 <head>
+	<meta charset="utf-8"/>
 	<title>GirlScript Foundation-Chennai Chapter | Achievements</title>
```
Comment (smaranjitghose) [inline achievements.html:5]: Make it GirlScript Website Boilerplate

File: achievements.html [hunk 6]
```diff
@@ -1,45 +1,45 @@
...
-	<meta charset="utf-8">
 	<meta content="width=device-width, initial-scale=1" name="viewport" />
 	
 	<meta name="keywords"
-		content="tech events, coding competition, introduction to coding, computer coding for beginners, girls who code program, coding contest, coding competition, girls can code, introduction to programming, learn computer programming">
+		content="tech events, coding competition, introduction to coding, computer coding for beginners, girls who code program, coding contest, coding competition, girls can code, introduction to programming, learn computer programming"/>
```
Comment (smaranjitghose) [inline achievements.html:9]: add more relevant ones like women in tech, open source

File: achievements.html [hunk 7]
```diff
@@ -1,45 +1,45 @@
...
 	<meta name="abstract"
-		content="This website is a base for Chennai chapter, and serves a template for other chapters. All the channel-specific tech events, coding competitions, and other resources are mentioned in this website.">
+		content="This website is a base for Chennai chapter, and serves a template for other chapters. All the channel-specific tech events, coding competitions, and other resources are mentioned in this website."/>
 	
-	<meta name="topic" content="GirlScript Foundation-Chennai Chapter | Achievements">
+	<meta name="topic" content="GirlScript Foundation-Chennai Chapter | Achievements"/>
 	
 	<meta name="summary"
-		content="This website is a base for Chennai chapter, and serves a template for other chapters. All the channel-specific tech events, coding competitions, and other resources are mentioned in this website.">
+		content="This website is a base for Chennai chapter, and serves a template for other chapters. All the channel-specific tech events, coding competitions, and other resources are mentioned in this website."/>
```
Comment (smaranjitghose) [inline achievements.html:30]: Change as stated above

File: achievements.html [hunk 8]
```diff
@@ -1,45 +1,45 @@
...
-	<meta name="copyright" content="GirlScript Foundation-Chennai Chapter | Achievements">
+	<meta name="designer" content="GirlScript Foundation-Chennai Chapter | Achievements"/>
+	<meta name="copyright" content="GirlScript Foundation-Chennai Chapter | Achievements"/>
 	
 	<meta name="abstract"
-		content="This website is a base for Chennai chapter, and serves a template for other chapters. All the channel-specific tech events, coding competitions, and other resources are mentioned in this website.">
+		content="This website is a base for Chennai chapter, and serves a template for other chapters. All the channel-specific tech events, coding competitions, and other resources are mentioned in this website."/>
 	
-	<meta name="topic" content="GirlScript Foundation-Chennai Chapter | Achievements">
+	<meta name="topic" content="GirlScript Foundation-Chennai Chapter | Achievements"/>
```
Comment (smaranjitghose) [inline achievements.html:27]: Change as stated above

File: achievements.html [hunk 9]
```diff
@@ -1,45 +1,45 @@
...
 	<meta name="AdsBot-Google" content="index,follow" />
 	
-	<meta name="designer" content="GirlScript Foundation-Chennai Chapter | Achievements">
-	<meta name="copyright" content="GirlScript Foundation-Chennai Chapter | Achievements">
+	<meta name="designer" content="GirlScript Foundation-Chennai Chapter | Achievements"/>
+	<meta name="copyright" content="GirlScript Foundation-Chennai Chapter | Achievements"/>
 	
 	<meta name="abstract"
-		content="This website is a base for Chennai chapter, and serves a template for other chapters. All the channel-specific tech events, coding competitions, and other resources are mentioned in this website.">
+		content="This website is a base for Chennai chapter, and serves a template for other chapters. All the channel-specific tech events, coding competitions, and other resources are mentioned in this website."/>
```
Comment (smaranjitghose) [inline achievements.html:25]: Change as stated above

File: achievements.html [hunk 10]
```diff
@@ -1,45 +1,45 @@
...
 	<meta name="msapplication-tooltip"
-		content="GirlScript Chennai is a website for the Chennai Chapter that showcases events and opportunities in Chennai.">
+		content="GirlScript Chennai is a website for the Chennai Chapter that showcases events and opportunities in Chennai."/>
 	
-	<meta name="apple-mobile-web-app-title" content="GirlScript Foundation-Chennai Chapter | Achievements">
+	<meta name="apple-mobile-web-app-title" content="GirlScript Foundation-Chennai Chapter | Achievements"/>
 	
-	<meta name="og:site_name" content="https://girlscriptboilerplate.netlify.app/achievements.html">
-	<meta name="og:title" content="GirlScript Foundation-Chennai Chapter | Achievements">
+	<meta name="og:site_name" content="https://girlscriptboilerplate.netlify.app/achievements.html"/>
```
Comment (smaranjitghose) [inline achievements.html:37]: Change as stated above

File: achievements.html [hunk 11]
```diff
@@ -1,45 +1,45 @@
...
+	<meta name="og:title" content="GirlScript Foundation-Chennai Chapter | Achievements"/>
```
Comment (smaranjitghose) [inline achievements.html:38]: Change as stated above

File: achievements.html [hunk 12]
```diff
@@ -1,45 +1,45 @@
...
 	
-	<meta name="og:url" content="https://girlscriptboilerplate.netlify.app/achievements.html">
+	<meta name="og:url" content="https://girlscriptboilerplate.netlify.app/achievements.html"/>
 	
-	<meta name="og:image" content="assets/Images/GirlScriptChennaiLogo.png">
+	<meta name="og:image" content="assets/Images/GirlScriptChennaiLogo.png"/>
```
Comment (smaranjitghose) [inline achievements.html:42]: Rename this asset as ```girlscript_website_logo```

**Comment by Castella1313** [comment]
@smaranjitghose should I make a new PR?

File: achievements.html (hunk 5 above)
Comment (Castella1313) [reply achievements.html:5]: @smaranjitghose the title? I'm so sorry if I am wrong, I am an absolute beginner, just trying to learn all these things.

**Comment by smaranjitghose** [comment]
You can make the changes in this one itself. Upto you

File: achievements.html (hunk 5 above)
Comment (smaranjitghose) [reply achievements.html:5]: Yes, please

**Comment by smaranjitghose** [comment]
@Castella1313 can you please fix the merge conflict

**Pull request merged**